app/src/main/java/com/beemdevelopment/aegis/otp/GoogleAuthInfo.java

```diff
@@ -343,5 +353,30 @@ public int getBatchIndex() {
         public int getBatchId() {
             return _batchId;
         }
+
+        public static List<Integer> getMissingIndexes(@NonNull List<Export> exports) throws IllegalArgumentException {
+            if (Objects.isNull(exports) || exports.isEmpty()) {
+                throw new IllegalArgumentException("Provided list was null or empty.");
```

Comment:
I don't think we need to raise an exception here. We already check whether the list is empty before calling, so something like this at the top of the function will suffice and is also correct:

```java
List<Integer> missingIndices = new ArrayList<>();
if (exports.isEmpty()) {
    return missingIndices;
}
```